kodadot/nft-gallery
# Captain planet supplemental release

- #3463 global offers
- #3463 adding option in navbar
- logged in check
- logged in check 2
- changes
- changes logic
- warnings fix
- fix_tests
- label fix
- Update components/bsx/Offer/MyOffer.vue
- Update components/bsx/Offer/MasterOfferTable.vue
- move carousel components into carousel
- remove unused async
- revert commented passionFeed
- remove unused async
- [skip netlify] Update Bumped minor version of packages
- feat: box-shadow for navbar
- fix: revert
- feat: make shadow same direction with dropdown
- feat: on light mode
- test 3d once
- test with another 3d once
- define GalleryItemCarousel for bsx
- patch vuex
- feat: using dropdown shadow
- Update styles/themes/_dark.scss
- Update styles/themes/_dark.scss

**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type


### What's new?

- [ ] Requires deployment <rubick/magick_issue>

### Before submitting Pull Request, please make sure:


### Optional


### Had issue bounty label?


### Community participation


### Screenshot
